packages/spark/components/_stepper.scss

```diff
@@ -40,6 +35,7 @@
 .sprk-c-Stepper__slider--active .sprk-c-Stepper__step-icon,
 .sprk-c-Stepper__slider--active .sprk-c-Stepper__step-description {
   opacity: 1;
+  transition: opacity $sprk-stepper-slider-text-fade-in-timing ease 0.2s;
```

Comment:
@afebbraro do you see any issue with changing this transition to *only* apply to the active slider? The point here is to fade in, but not out. If you look at the way vanilla operates, the text and title of the stepper only ever **fade-in**, and so this change to the css technically does not change anything about how that interaction is rendered. If anything there's now just fewer unintended side effects.

Reply:
Hmm, I actually didn't write the sprk-c-Stepper__slider--active class name and I think I had it on the selected step originally. But, scoping the transition in this way seems fine to do and like you said doesn't change anything about how it works currently. So, if it solves some weird React bug then I don't see harm in doing this, especially like you said, if it adds more predicability. Let's do it.

Reply:
Oh! But can you make the delay a var? @bobwise. The hard-coded 0.2s should be placed as var in our settings file.